<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rms_updateClass</class>
 <widget class="QMainWindow" name="rms_updateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rms_update</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string>O</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Md5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Progress</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck"/>
   <addaction name="actionUpdate"/>
   <addaction name="separator"/>
   <addaction name="actionMake"/>
   <addaction name="actionSetting"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCheck">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/check.png</normaloff>:/images/check.png</iconset>
   </property>
   <property name="text">
    <string>check</string>
   </property>
   <property name="toolTip">
    <string>check update package</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
   </property>
   <property name="text">
    <string>update</string>
   </property>
   <property name="toolTip">
    <string>update package</string>
   </property>
  </action>
  <action name="actionMake">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/xml.png</normaloff>:/images/xml.png</iconset>
   </property>
   <property name="text">
    <string>Make</string>
   </property>
   <property name="toolTip">
    <string>make update plist </string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/setting.png</normaloff>:/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>setting</string>
   </property>
   <property name="toolTip">
    <string>setting</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="rms_update.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="toolTip">
    <string>about us...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rms_update.qrc"/>
 </resources>
 <connections/>
</ui>
